test/selenium/ContributorSpec.scala

```diff
@@ -72,7 +72,7 @@ class ContributorSpec extends FeatureSpec with Browser with GivenWhenThen with B
     scenario("Monthly contribution sign-up with PayPal - USD") {
 
       val landingPage = ContributionsLanding("us")
-      val expectedPayment = "10.00"
+      val expectedPayment = "15.00"
```

Comment:
I'm not sure if hardcoding this here is sensible in retrospect...

Reply:
It's fine. If someone changes the default in future the tests will actually be running and they'll notice that they've failed.